### PR TITLE
unexport all methods

### PR DESCRIPTION
**Pull request by Guitarbum722**
* rename interface and create one method
* unexport methods on `Align`

### CODE REVIEW

File: align_test.go [hunk 1]
```diff
@@ -191,38 +191,38 @@ Karleigh,Destiny,Dean,nunc.In@lorem.edu,Stockholms län,Märsta,9038,Shaine Reil
...
 
-	sw := NewAligner(strings.NewReader(input), os.Stdout, comma, TextQualifier{On: false})
+	a := newAlign(strings.NewReader(input), os.Stdout, comma, TextQualifier{On: false})
 
 	b.ReportAllocs()
 	b.ResetTimer()
 
 	for i := 0; i < b.N; i++ {
-		_ = sw.ColumnCounts()
+		_ = a.columnLength()
```
Comment (briandowns) [inline align_test.go]: You can just call the method without the need to ignore output with the `_`.

File: align_test.go [hunk 2]
```diff
@@ -191,38 +191,38 @@ Karleigh,Destiny,Dean,nunc.In@lorem.edu,Stockholms län,Märsta,9038,Shaine Reil
...
 
-	sw := NewAligner(strings.NewReader(input), os.Stdout, comma, TextQualifier{On: true, Qualifier: "\""})
+	a := newAlign(strings.NewReader(input), os.Stdout, comma, TextQualifier{On: true, Qualifier: "\""})
 
 	b.ReportAllocs()
 	b.ResetTimer()
 
 	for i := 0; i < b.N; i++ {
-		_ = sw.SplitWithQual(input, comma, "\"")
+		_ = a.splitWithQual(input, comma, "\"")
```
Comment (briandowns) [inline align_test.go]: Same as above.

File: align_test.go [hunk 3]
```diff
@@ -191,38 +191,38 @@ Karleigh,Destiny,Dean,nunc.In@lorem.edu,Stockholms län,Märsta,9038,Shaine Reil
...
 
-	sw := NewAligner(strings.NewReader(input), os.Stdout, comma, TextQualifier{On: false})
+	a := newAlign(strings.NewReader(input), os.Stdout, comma, TextQualifier{On: false})
 
 	b.ReportAllocs()
 	b.ResetTimer()
 
 	for i := 0; i < b.N; i++ {
-		_ = sw.SplitWithQual(input, comma, "\"")
+		_ = a.splitWithQual(input, comma, "\"")
```
Comment (briandowns) [inline align_test.go]: Same as above.

File: align.go [hunk 4]
```diff
@@ -275,8 +264,8 @@ func trailingPad(s string, padLen int) string {
 	return s + string(pad)
 }
 
-// SplitWithQual basically works like the standard strings.Split() func, but will consider a text qualifier if set.
-func (a *Aligner) SplitWithQual(s, sep, qual string) []string {
+// splitWithQual basically works like the standard strings.Split() func, but will consider a text qualifier if set.
+func (a *Align) splitWithQual(s, sep, qual string) []string {
```
Comment (briandowns) [inline align.go]: Remove blank line.

File: align_test.go (hunk 1 above)
Comment (Guitarbum722) [inline align_test.go]: Good call - thanks!  Done.

File: align.go (hunk 4 above)
Comment (Guitarbum722) [inline align.go]: Done!

**Pull request merged**